### DISCUSSION TITLE
Inviare mail ai comuni in zona rossa

### DISCUSSION THREAD

**Discussion by dennisangemi** [opendatasicilia/informa-covid19#60] [Idee/Proposte]
Buongiorno a tutti/e,
data la scarsa ricezione di informacovid da parte delle amministrazioni comunali, perchè non inviare una mail ai comuni che attualmente sono in zona rossa? Dovrebbero essere altamente interessati data la criticità del loro stato e grazie al [lavoro ](https://tinyurl.com/zonerossesicilia)di @pigreco sappiamo di quali comuni si tratta.

Ho predisposto un[ testo per email ](https://docs.google.com/document/d/1CVTbb6S_UPGiq448aRTqon6IzF_CyUyMTRGCbl29fUA/edit?usp=sharing). 
Che ne dite?

**Comment by gpirrotta** [comment]
Ciao Dennis e grazie, mi sembra un'ottima idea.
In questo momento in mappa ci sono 4 comuni in zona rossa per cui l'invio delle email risulterebbe abbastanza gestibile.
Per l'inoltro chiediamo a @picomiles.

**Comment by pigreco** [comment]
@dennisangemi sei un vulcano, hai ottime idee!!!

**Reply by dennisangemi** [reply]
Credimi, è il "nenticchiffari"

**Comment by dennisangemi** [comment]
@picomiles ha inviato mail. Grazie!

**Reply by picomiles** [reply]
Confermo, di nulla.
Mailchimp consente di avere un minimo di feedback dalle email inoltrate; vi aggiorno

**Comment by aborruso** [comment]
Grazie

Il mar 20 lug 2021, 16:45 Nino Galante ***@***.***> ha
scritto:
